_ja/overviews/core/futures.md

```diff
@@ -139,43 +139,18 @@ Future の実装の多くは、Future の結果を知りたくなったクライ
     }
 
 `onComplete` メソッドは、Future 計算の失敗と成功の両方の結果を扱えるため、汎用性が高い。
-成功した結果のみ扱う場合は、(部分関数を受け取る) `onSuccess` コールバックを使う:
+成功した結果のみ扱う場合は、`foreach` コールバックを使う:
```

Comment:
Kind of odd to call `foreach` a _callback_ in Scala terminology, but I guess it's on https://github.com/scala/docs.scala-lang/pull/1051.